pkg/collector/corechecks/snmp/internal/test/test_profiles/validation_error.yaml

```diff
@@ -1,7 +1,5 @@
 # Profile for F5 BIG-IP devices
 #
-extends:
-  - does_not_exist.yaml
```

Comment:
why make this change ?

Reply:
This is related to this change here https://github.com/DataDog/datadog-agent/pull/19835/files#r1350283384

after this change, the test was failing due to extend entry "does not exist" instead of "invalid regex" at line 16.